config/ores.yaml

```diff
@@ -0,0 +1,85 @@
+application_root: "/scores"
+url_prefix: "/scores"
+
+# Scorers
+scorers:
+  enwiki:
+    extractor: enwiki_api
+    models:
+      reverted: enwiki_revert
+  ptwiki:
+    extractor: ptwiki_api
+    models:
+      reverted: ptwiki_revert
+  fawiki:
+    extractor: fawiki_api
+    models:
+      reverted: fawiki_revert
+#  trwiki:
+#    extractor: trwiki_api
+#    models:
+#      reverted: trwiki_revert 
+  frwiki:
+    extractor: frwiki_api
+    models:
+      reverted: frwiki_revert
+#  simplewiki:
+#    extractor: simplewiki_api
+#    models:
+#      reverted: simplewiki_revert
+
+# Feature extractors
+extractors:
+  defaults:
+    class: revscoring.extractors.APIExtractor
+    user_agent: Object Revision Evaluation Service
+  enwiki_api:
+    url: http://en.wikipedia.org/w/api.php
+    language: english
+  ptwiki_api:
+    url: http://pt.wikipedia.org/w/api.php
+    language: portuguese
+  fawiki_api:
+    url: https://fa.wikipedia.org/w/api.php
+    language: persian
+  trwiki_api:
+    url: https://tr.wikipedia.org/w/api.php
+    language: turkish
+  frwiki_api:
+    url: https://fr.wikipedia.org/w/api.php
+    language: french
+  simplewiki_api:
```

Comment:
Negative.  That way, once those models are ready, we only have to remove one set of comments.